submission/distributor_test.go

```diff
@@ -36,41 +33,34 @@ import (
 	"github.com/google/certificate-transparency-go/x509"
 	"github.com/google/certificate-transparency-go/x509util"
 
-	"io/ioutil"
+	"github.com/google/go-cmp/cmp"
 )
 
-// ReadPEMFile returns the first certificate it finds in pemFile, as base64-encoded DER
-func readPEMFile(pemFile string) string {
-	pemCerts, _ := ioutil.ReadFile(pemFile)
-	for len(pemCerts) > 0 {
-		var block *pem.Block
-		block, pemCerts = pem.Decode(pemCerts)
-		if block == nil {
-			break
-		}
-		if block.Type != "CERTIFICATE" || len(block.Headers) != 0 {
-			continue
-		}
-		return base64.StdEncoding.EncodeToString(block.Bytes)
+// ReadCertFile returns the first certificate it finds in file provided.
+func readCertFile(filename string) string {
+	fmt.Println(filename)
```

Comment:
I expect the above line was just there for debugging during development, and should now be removed?